apps/qubit/modules/jobs/templates/reportSuccess.php

```diff
@@ -48,26 +48,30 @@
 
   <?php if ($job->getObjectModule() && $job->getObjectSlug()): ?>
     <div class="job-report-field">
-      <div><?php echo __('Associated item') ?></div>
+      <div><?php echo __('Associated record') ?></div>
       <div><?php echo link_to(__('Link'), array('module' => $job->getObjectModule(), 'slug' => $job->getObjectSlug())) ?></div>
     </div>
   <?php endif; ?>
 
   <?php if ($job->downloadPath): ?>
     <div class="job-report-field">
       <div><?php echo __('Download path') ?></div>
-      <div><?php echo link_to(__('Link'), public_path($job->downloadPath)) ?>
+      <div><?php echo link_to(__('Link'), public_path($job->downloadPath)) ?></div>
     </div>
   <?php endif; ?>
 </section>
 
 <section id="log-area">
   <h2><?php echo __('Log') ?></h2>
-  <div id="job-log-output">
-    <?php echo render_value($job->output) ?>
+  <div>
+    <pre id="job-log-output"><?php echo render_value(trim($job->output)) ?></pre>
   </div>
 </section>
 
-<div id="job-return-link">
-  <?php echo link_to(__('« Return to jobs management page'), array('module' => 'jobs', 'action' => 'browse')) ?>
-</div>
+<section class="actions" id="job-return-link">
```

Comment:
Almost, but you need to take the `actions` out of the main `#contents`. We do that with slots, see for example: https://github.com/artefactual/atom/blob/qa/2.3.x/plugins/qtAccessionPlugin/modules/accession/templates/indexSuccess.php. Once you do that change, you shouldn't need `#job-return-link`.